src/common.ts

```diff
@@ -1,58 +1,109 @@
-import { SNSEvent } from "aws-lambda";
 import { CloudWatch } from "aws-sdk";
-import { metricTimestampFromAlarmEvent, alarmNameFromAlarmEvent, isAlarmEventForState } from "./cloudwatchAlarmSnsEvent";
-import { AlarmState, secondsBetween } from "./alarmHistory";
+import { AlarmState, secondsBetweenFromHistory, secondsBetweenPreviouseState } from "./alarmHistory";
+import {
+  alarmNameFromAlarmEvent, isAlarmEventForState,
+  metricTimestampFromAlarmEvent,
+} from "./cloudwatchAlarmEvent";
+
+export interface CloudwatchStateChangeEvent {
```

Comment:
Isn't this type already available - in `aws-sdk` or `aws-lambda`?

Reply:
I couldn't find a type for it, that's why I created one